landlab/plot/landslides/plot_landslides.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
@ChristinaB these functions are not yet in the docs. new/modified rst files are needed.